src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -182,13 +240,19 @@
             [button {:on-click #(dispatch [:modal/toggle])
                      :title "Choose Database"}
              [:> LibraryBooks]]
+            [separator]
+            ;; obnoxious line separator
+            [button {:on-click #(js/alert "help popup") :title "Help Popup"}
```

Comment:
this looks like it opens a javascript alert, not the actual help-modal.

Reply:
That is correct, I am stuck, and this is my temporary "save point".